bin/dockerfilelint

```diff
@@ -4,14 +4,36 @@ var process = require('process');
 var fs = require('fs');
 var os = require("os");
 var path = require('path');
-var argv = require('yargs').argv;
+var usage = 'Usage: dockerfilelint [files | content..] [options]';
+var argv = require('yargs')
+             .usage(usage)
```

Comment:
Sorry to nit-pick, but could we reduce the indentation from here to line 29 so it starts at 2 spaces? I'm so used to [standard js](https://standardjs.com/) that this bugs me.

Aside: we should really add a code linting npm script to package.json, but that's outside the scope of this PR.